std/typecons.d

```diff
@@ -1859,7 +1859,7 @@ unittest
   Returns:
       A string to be mixed in to an aggregate, such as a `struct` or `class`.
 */
-string alignForSize(E...)(string[] names...)
+string alignForSize(E...)(const char[][] names...)
```

Comment:
Isn't the question rather why it should be restricted to `string` in the first place?